### DISCUSSION TITLE
🍬 Tell me your favourite candies and/or snacks 🍭

### DISCUSSION THREAD

**Discussion by b08902094** [NAIST-SE/AttractingContributors#1083] [2024 Attraction Contributions]
Please make a PR and tell me your favourite sweets/snacks 😋 
[Repo URL](https://github.com/b08902094/AttractingContributors-Baking)

**Comment by kozmr** [comment]
Hi, I contributed to your project.
Could you contribute to my project?

https://github.com/kozmr/AttractingContributors-YouTube
Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1113

**Comment by kohanaist** [comment]
I have made a PR on your repo. 
Could you contribute to mine? 
https://github.com/kohanaist/AttractingContributors-technical-or-specialized-book

**Comment by anzhitingg** [comment]
Hello！
I have just made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/anzhitingg/AttractingContributors-MBTI
Thank you very much！
